### ISSUE TITLE
cannot stat 'OpenRA/mods/modcontent': No such file or directory

### ISSUE THREAD

**Issue by fusion809** [OpenRA/OpenRA#21628]
## Issue Summary
When packaging on Arch Linux, commit c6af758 fails to package with the error:

```bash
Installing engine to /path/to/openra-wts-git/pkg/openra-wts-git/usr/lib/openra
Installing engine files to /path/to/openra-wts-git/pkg/openra-wts-git/usr/lib/openra
Installing common mod files to /path/to/openra-wts-git/pkg/openra-wts-git/usr/lib/openra
Installing mod cnc to /path/to/openra-wts-git/pkg/openra-wts-git/usr/lib/openra
cp: cannot stat '/path/to/openra-wts-git/src/OpenRA/mods/modcontent': No such file or directory
```

This error comes from the command `make prefix=/usr DESTDIR="$pkgdir" install DEBUG=false RUNTIME=mono`.

## Reproduction
On an up-to-date Arch Linux install, or Arch Linux WSL, download the AUR package openra-wts-git and build it with the `makepkg -s` command. You should experience this error.

## Expected behaviour
I expected `make prefix=/usr DESTDIR="$pkgdir" install DEBUG=false RUNTIME=mono` to run without error and move the appropriate files to $pkgdir.

**Comment by PunkPun** [comment]
likely regression from #21623

**Comment by ottelo9** [comment]
I think its fixed https://github.com/OpenRA/OpenRA/pull/21629

**Comment by fusion809** [comment]
> I think its fixed #21629

It is, I can confirm.
